### PR TITLE
Service exposition docs

### PR DESCRIPTION
**Pull request by fhennig**
# Description

*Please add a description here. This will become the commit message of the merge request later.*

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

### CODE REVIEW

File: rust/crd/src/lib.rs [hunk 1]
```diff
@@ -169,17 +169,17 @@ pub struct AirflowClusterConfig {
     pub expose_config: Option<bool>,
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub load_examples: Option<bool>,
-    /// In the future this setting will control, which ListenerClass <https://docs.stackable.tech/home/stable/listener-operator/listenerclass.html>
-    /// will be used to expose the service.
-    /// Currently only a subset of the ListenerClasses are supported by choosing the type of the created Services
-    /// by looking at the ListenerClass name specified,
-    /// In a future release support for custom ListenerClasses will be introduced without a breaking change:
+    /// This enum controls which type of Service the Operator creates for this AirflowCluster:
```
Comment (maltesander) [inline rust/crd/src/lib.rs:172]: Enum is the rust/programming name, this is basically user facing doc, i would say `field` instead of `enum` ?

Comment (fhennig) [reply rust/crd/src/lib.rs]: I changed it!

**Review by maltesander — APPROVED** [review]
LGTM!

**Review by maltesander — DISMISSED** [review]
LGTM!

**Review by maltesander — APPROVED** [review]
LGTM!

**Pull request merged**